### PR TITLE
Adds placeholders for frontpage

### LINKED ISSUE #289: Empty states
When there are no events, articles, editions of Offline etc, it's nice to have a quick sentence saying "yo there aren't any events y'all", or "oopsie woopsie, we couldn't find any articles" and so on. I prefer having an empty state to just removing the section if it has no content, because then you're still reminded that the sections exist, and how that plays into the design.
It's mostly for developing locally. Since articles and Offline-editions don't have an expiration date, they shouldn't normally be empty in prod. But who knows. Low priority anyways.

### PR DESCRIPTION
**Pull request by VigdisIS**
Fixes #289 .

Adds placeholders for articles, Offline and list-view.

Skjermbilde 2019-10-02 kl  19 47 08
Skjermbilde 2019-10-02 kl  19 47 59
Skjermbilde 2019-10-02 kl  19 47 30

### CODE REVIEW

File: src/events/components/ListView/ListEventPlaceholder.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { faCalendarAlt, faUser } from '@fortawesome/free-regular-svg-icons/';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import classnames from 'classnames';
+import React from 'react';
+import { EventTypeEnum, getEventColor, getEventType } from '../../models/Event';
```
Comment (henrikhorluck) [inline src/events/components/ListView/ListEventPlaceholder.tsx]: https://github.com/dotkom/onlineweb-frontend/wiki/(Opinionated)-Structuring-and-ordering-of-imports

File: src/frontpage/components/Articles/MainArticlePlaceholder.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import classnames from 'classnames';
+import React from 'react';
+import { DefaultEventImage } from '../../../events/components/DefaultEventImage';
```
Comment (henrikhorluck) [inline src/frontpage/components/Articles/MainArticlePlaceholder.tsx]: https://github.com/dotkom/onlineweb-frontend/wiki/(Opinionated)-Structuring-and-ordering-of-imports

File: src/events/components/ListView/list.less [hunk 3]
```diff
@@ -96,12 +103,15 @@
...
   }
+
   .icon {
     font-size: 15px;
   }
+
   .suppText {
     font-size: 12px;
   }
+
```
Comment (henrikhorluck) [inline src/events/components/ListView/list.less]: Uhm, was this from prettier?

File: src/frontpage/components/Articles/SmallArticlePlaceholder.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import classnames from 'classnames';
+import React from 'react';
+import { DefaultEventImage } from '../../../events/components/DefaultEventImage';
```
Comment (henrikhorluck) [inline src/frontpage/components/Articles/SmallArticlePlaceholder.tsx]: https://github.com/dotkom/onlineweb-frontend/wiki/(Opinionated)-Structuring-and-ordering-of-imports

File: src/frontpage/components/Articles/articles.less [hunk 5]
```diff
@@ -83,6 +91,7 @@
   .smallArticle {
     grid-template-columns: 1fr;
   }
+
```
Comment (henrikhorluck) [inline src/frontpage/components/Articles/articles.less]: Same as above

File: src/frontpage/components/Offline/OfflineCarouselPlaceholder.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import classnames from 'classnames';
+import { IRefObject } from 'common/hooks/useRefMap';
+import React from 'react';
+import { DefaultEventImage } from '../../../events/components/DefaultEventImage';
```
Comment (henrikhorluck) [inline src/frontpage/components/Offline/OfflineCarouselPlaceholder.tsx]: https://github.com/dotkom/onlineweb-frontend/wiki/(Opinionated)-Structuring-and-ordering-of-imports

File: src/events/components/ListView/list.less (hunk 3 above)
Comment (VigdisIS) [reply src/events/components/ListView/list.less]: Yes.

File: src/frontpage/components/Articles/articles.less (hunk 5 above)
Comment (VigdisIS) [reply src/frontpage/components/Articles/articles.less]: Yes.

File: src/events/components/ListView/ListEventPlaceholder.tsx (hunk 1 above)
Comment (VigdisIS) [reply src/events/components/ListView/ListEventPlaceholder.tsx]: This is fixed.

File: src/events/components/ListView/index.tsx [hunk 7]
```diff
@@ -42,13 +42,21 @@ export const ListView = ({ filtered }: IProps) => {
...
   return (
     <>
-      <div className={style.grid}>
-        {displayEvents.map((event) => (
-          <Link to={`/events/${event.id}`} key={event.id}>
-            <ListEvent {...event} />
-          </Link>
-        ))}
-      </div>
+      {eventList[0] ? (
```
Comment (henrikhorluck) [inline src/events/components/ListView/index.tsx]: This is likely to cause an exception, so a better solution here, would be
```javascript
eventList.length !== 0 // or simply eventList.length
```

**Comment by FluidSense** [comment]
@VigdisIS @henrikhorluck 
Will this be revisited? Is it still relevant?

**Comment by FluidSense** [comment]
I'm closing this as stale, due to last commit date and no respond. Feel free to reopen if it is being revisited!

**Pull request closed without merging**